src/main/java/lotto/domain/LottoRankingSystem.java

```diff
@@ -0,0 +1,37 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum LottoRankingSystem {
+    FIFTH(3, 5000),
```

Comment:
```suggestion
    FIFTH(3, 5_000),
```
숫자를 표기할때는 3자리간격으로 `_`로 끊어주는 것이 가독성을 올려줍니다 😄